src/core/document.js

```diff
@@ -894,6 +901,7 @@ class PDFDocument {
       IsAcroFormPresent: this.formInfo.hasAcroForm,
       IsXFAPresent: this.formInfo.hasXfa,
       IsCollectionPresent: !!this.catalog.collection,
+      AreSignaturesPresent: this.formInfo.hasSignatures,
```

Comment:
For consistency with the other properties, since this is mostly used internally anyway, can we please call this `IsSignaturesPresent` instead (despite that being grammatically wrong)?

Also, please add the new property to the *existing* tests in https://github.com/mozilla/pdf.js/blob/master/test/unit/api_spec.js